azure-setup.sh

```diff
@@ -229,15 +229,17 @@ if [ "$terms_rsp" != "true" ]; then
     echo $mkt_rsp
 fi
 
-echo "Creating event subscription for an Azure subscription"
+echo "Creating event subscription for an Azure subscription, with enabled authorization"
 az eventgrid event-subscription create \
     --source-resource-id "/subscriptions/${sub_id}" \
     --name "$EVENT_SUB_NAME" \
     --endpoint "$webhook_endpoint" \
     --included-event-types \
     Microsoft.Resources.ResourceWriteSuccess \
     Microsoft.Resources.ResourceDeleteSuccess \
-    Microsoft.Resources.ResourceActionSuccess
+    Microsoft.Resources.ResourceActionSuccess \
```

Comment:
Don't we need --endpoint-type webhook?

Reply:
--endpoint-type   : The type of the destination endpoint. Default: webhook.